libs/execution_context/ebpf_program.c

```diff
@@ -225,9 +226,28 @@ _ebpf_program_free(ebpf_core_object_t* object)
 }
 
 static const ebpf_program_type_t*
-_ebpf_program_get_program_type(_In_ const ebpf_core_object_t* object)
+_ebpf_program_get_program_type_uuid(_In_ const ebpf_core_object_t* object)
 {
-    return ebpf_program_type((const ebpf_program_t*)object);
+    return ebpf_program_type_uuid((const ebpf_program_t*)object);
+}
+
+static const bpf_prog_type_t
+_ebpf_program_get_program_type(_In_ const ebpf_program_t* program)
```

Comment:
make return type and function name be consistent (prog_type vs program_type)

Reply:
changed function name.